packages/compass-e2e-tests/index.ts

```diff
@@ -118,8 +121,10 @@ async function setup() {
 
   fs.mkdirSync(LOG_PATH, { recursive: true });
 
-  debug('Getting mongodb server info');
-  updateMongoDBServerInfo();
+  if (!ATLAS_DOMAIN) {
+    debug('Getting mongodb server info');
+    updateMongoDBServerInfo();
```

Comment:
Various checks in tests rely on this information existing, you can't just skip it

Reply:
Ah I think I need to update the function to use the variable instead of hardcoded address like it currently does

Reply:
Yeah, probably can use DEFAULT_CONNECTION_STRING_1 there insteasd of just port, this is aligned more with what's expected

Reply:
[9585f50](https://github.com/mongodb-js/compass/pull/6381/commits/9585f5043155bec8d7c0ff6d30e5f409d809556c)